draft-ounsworth-cfrg-kem-combiners.mkd

```diff
@@ -258,35 +261,27 @@ The need for a KEM-based authenticated key establishment arises, for example, wh
 # KEM Combiner construction {#sec-kem-combiner}
 
 <!-- TODO: read and cite the ETSI doc "Quantum-safe Hybrid Key Exchanges"
-https://www.etsi.org/deliver/etsi_ts/103700_103799/103744/01.01.01_60/ts_103744v010101p.pdf
+https://www.etsi.org/deliver/etsi_ts/103700_103799/103744/01.01.01_60/ts_103744v010101p.pdf -->
 
 TODO: as per https://www.enisa.europa.eu/publications/post-quantum-cryptography-integration-study section 4.2, might need to specify behaviour in light of KEMs with a non-zero failure probability.
--->
-
 
-A KEM combiner is a function that takes in two or more shared secrets `ss_i` and returns a combined shared secret `ss`.
-
-~~~
-ss = kemCombiner(ss_1, ss_2, ..., ss_n)
-~~~
+A KEM combiner is a function that takes the output of two or more KEM encapsulations and combines them to produce a single shared secret.
 
 This document assumes that shared secrets are the output of a KEM, but without loss of generality they MAY also be any other source of cryptographic key material, such as pre-shared keys (PSKs), with PQ/PSK being a quantum-safe migration strategy being made available by some protocols, see for example IKEv2 in [RFC8784].
 
 In general it is desirable to use a split-key PRF as a KEM combiner, meaning that the combiner has the properties of a PRF when keyed by any of its single inputs.
 The following simple yet generic construction can be used in all IETF protocols that need to combine the output of two or more KEMs:
 
 ~~~
-ss = KDF(counter || k_1 || ... || k_n || fixedInfo, outputBits)
+ss = KDF(k_1 || ... || k_n, fixedInfo)
 ~~~
 {: #tab-kemCombiner title="general KEM combiner construction" }
 
 where:
 
-- `KDF` represents a suitable choice of a cryptographic key derivation function,
+- `KDF` represents a suitable choice of a cryptographic key derivation function as defined in {{sec-instantiation}}.
 - `k_i` represent the constant-length input keys and is discussed in {{sec-k_i-construction}},
 - `fixedInfo` is some protocol-specific KDF binding,
-- `counter` parameter is instantiation-specific and is discussed in {{sec-instantiation}}.
-- `outputBits` determines the length of the output keying material,
```

Comment:
Line 411 includes the outputBits parameter, but it has been removed here.  My guess is that you would want to include it.

Reply:
Good catch.